packages/node_modules/@node-red/nodes/core/network/22-websocket.js

```diff
@@ -46,6 +46,7 @@ module.exports = function(RED) {
 
         // Store local copies of the node configuration (as defined in the .html)
         node.path = n.path;
+        node.subprotocol = n.subprotocol; // optional client protocol
```

Comment:
I see the WS protocol lets you provide either a string or array of strings. Should we support the user providing a comma-separated list of values that we parse to an array?

```suggestion
        if (typeof n.subprotocol === "string") {
            // Split the string on comma and trim each result
            node.subprotocol = subprotocol.split(",").map(v => v.trim())
        } else {
            node.subprotocol = [];
        }
```